AlphaWalletTests/Transfer/Controllers/TransactionConfiguratorTests.swift

```diff
@@ -32,11 +32,11 @@ class TransactionConfiguratorTests: XCTestCase {
     }
     
     func testLoadEtherConfiguration() {
-        let configurator = TransactionConfigurator(session: .make(), account: .make(), transaction: .make(gasLimit: BigInt(90000), gasPrice: .none))
+        let configurator = TransactionConfigurator(session: .make(), account: .make(), transaction: .make(gasLimit: BigInt(21000), gasPrice: .none))
 
         configurator.load { _ in }
         
         XCTAssertEqual(BigInt(GasPriceConfiguration.defaultPrice), configurator.configuration.gasPrice)
-        XCTAssertEqual(BigInt(90000), configurator.configuration.gasLimit)
+        XCTAssertEqual(BigInt(21000), configurator.configuration.gasLimit)
```

Comment:
Looks like the problem is here instead, in `TransactionConfigurator.load(completion:)`:

```
if gasLimitNotSet {
	estimateGasLimit()
}
```

The old version only continues with the rest of the function if the gas limit has not been set:

```
guard gasLimitNotSet else {
	return completion(.success(()))
}
estimateGasLimit()
```

Reply:
While we are at it, do you want to make the gas limit value in this test case a number that is not `90000` and `21000`? Easier to detect when it's failing, because those 2 numbers are used in `GasLimitConfiguration`.